bedrock/mozorg/templates/mozorg/about/policy/patents/index.html

```diff
@@ -82,7 +82,7 @@ <h2 data-accordion-role="tab">{{ _('Does the MOSPL affect any other non-exclusiv
         <h2 data-accordion-role="tab">{{_('How does this project address the software patent quality issues?') }}</h2>
         <div data-accordion-role="tabpanel">
         {% trans guide=url('mozorg.about.policy.patents.guide') %}
-          <p>The more innovations in open source projects are patented, the easier it will be for patent examiners to identify them as prior art for future applications. This will lead to higher quality software patents. We do not predict that Mozilla’s own patents and defensive publications will, by themselves, be enough to substantially increase the quality of every type of issued patent. However, we hope that the example this project sets along with the <a href="{{ guide }}">Open Software Patent Licensing Guide</a> we’ve provided will encourage others to consider openly licensing their portfolio and applying for patents on their open source innovations to help increase software patent quality. Mozilla has also begun to advocate publicly for patent law reform that would further advance software patent quality: <a rel="external" target='_blank' href="https://blog.mozilla.org/netpolicy/files/2015/05/Mozilla-USPTO-Filing-May-2015-5-6.pdf">https://blog.mozilla.org/netpolicy/files/2015/05/Mozilla-USPTO-Filing-May-2015-5-6.pdf</a>.</p>
+          <p>The more innovations in open source projects are patented, the easier it will be for patent examiners to identify them as prior art for future applications. This will lead to higher quality software patents. We do not predict that Mozilla’s own patents and defensive publications will, by themselves, be enough to substantially increase the quality of every type of issued patent. However, we hope that the example this project sets along with the <a href="{{ guide }}">Open Software Patent Licensing Guide</a> we’ve provided will encourage others to consider openly licensing their portfolio and applying for patents on their open source innovations to help increase software patent quality. Mozilla has also begun to advocate publicly for patent law reform that would further advance software patent quality: <a rel="external noopener noreferrer" target='_blank' href="https://blog.mozilla.org/netpolicy/files/2015/05/Mozilla-USPTO-Filing-May-2015-5-6.pdf">https://blog.mozilla.org/netpolicy/files/2015/05/Mozilla-USPTO-Filing-May-2015-5-6.pdf</a>.</p>
```

Comment:
There's no real reason this link really needs `target="_blank"` either, please remove.